draft-packages/popover/KaizenDraft/Popover/styles.scss

```diff
@@ -49,12 +45,13 @@ $large-width: 450px;
 }
 
 %box {
-  background: white;
+  background: $color-white;
   border: $border-solid-border-width $border-solid-border-style $color-gray-300;
   filter: drop-shadow(0 0 7px rgba(0, 0, 0, 0.1));
   border-radius: $border-solid-border-radius;
   color: $color-purple-800;
   text-align: left;
+  padding: $spacing-sm 1rem 1rem;
```

Comment:
nit: I don't like how the UI Kit is using spacing that isn't in our variables 😬

Reply:
The spacing is 12px in the UI kit but because of the line-height of the heading and body text, adding this 1rem padding around the whole container was the workaround to still getting a total spacing of 12px rather than adding additional padding in multiple spots. If that makes any sense.